src/main/java/TaskList.java

```diff
@@ -0,0 +1,55 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> toDoList = new ArrayList<>();
```

Comment:
Try not to do type inference when creating the ArrayList. Consider declaring the type explicitly instead:
```private ArrayList<Task> toDoList = new ArrayList<Task>();```